ci/jenkins/Jenkins_steps.groovy

```diff
@@ -1736,17 +1736,6 @@ def sanity_lint() {
     }]
 }
 
-def sanity_rat_license() {
```

Comment:
Please focus and don't just change stuff while you're at it and stumble across.

Reply:
Put some consideration in first - don't just remove something because you don't understand it but do your due diligence. 

You will notice soon that this was a mistake. Please be careful and focus a PR on a single purpose to reduce the blast radius.

Reply:
@vexilligera Marco's point is that while the RAT test is not run anymore in the nightly Jenkinsfile, and you can thus ignore the "RAT" parts of the nightly Dockerfile, the RAT test is run on every PR, so you shouldn't delete this function

Reply:
Added back.